File: client/src/main/java/com/sio/pizzeria/DTO/HistoriqueTableDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.sio.pizzeria.DTO;

/**
 *
 * @author Lenovo P50
 */
public class HistoriqueTableDTO {
    	
    private int idHistoriqueTable;
    private int historiqueNumeroTable;
    private int historiqueNbPersonne;

    public HistoriqueTableDTO(int historiqueNumeroTable, int historiqueNbPersonne) {
        this.historiqueNumeroTable = historiqueNumeroTable;
        this.historiqueNbPersonne = historiqueNbPersonne;
    }

    public int getIdHistoriqueTable() {
        return idHistoriqueTable;
    }

    public void setIdHistoriqueTable(int idHistoriqueTable) {
        this.idHistoriqueTable = idHistoriqueTable;
    }

    public int getHistoriqueNumeroTable() {
        return historiqueNumeroTable;
    }

    public void setHistoriqueNumeroTable(int historiqueNumeroTable) {
        this.historiqueNumeroTable = historiqueNumeroTable;
    }

    public int getHistoriqueNbPersonne() {
        return historiqueNbPersonne;
    }

    public void setHistoriqueNbPersonne(int historiqueNbPersonne) {
        this.historiqueNbPersonne = historiqueNbPersonne;
    }

    
}

File: server/DTO/HistoriqueTableDTO.php
<?php


class HistoriqueTableDTO implements JsonSerializable
{
    private $idHistoriqueTable;
    private $historiqueNumeroTable;
    private $historiqueNbPersonne;


    public function __construct($historiqueNumeroTable, $historiqueNbPersonne)
    {
        $this->historiqueNumeroTable = $historiqueNumeroTable;
        $this->historiqueNbPersonne = $historiqueNbPersonne;
    }

    public function getIdHistoriqueTable()
    {
        return $this->idHistoriqueTable;
    }

    public function setIdHistoriqueTable($idHistoriqueTable)
    {
        $this->idHistoriqueTable = $idHistoriqueTable;
    }

    public function getHistoriqueNumeroTable()
    {
        return $this->historiqueNumeroTable;
    }

    public function setHistoriqueNumeroTable($historiqueNumeroTable)
    {
        $this->historiqueNumeroTable = $historiqueNumeroTable;
    }

    public function getHistoriqueNbPersonne()
    {
        return $this->historiqueNbPersonne;
    }

    public function setHistoriqueNbPersonne($historiqueNbPersonne)
    {
        $this->historiqueNbPersonne = $historiqueNbPersonne;
    }

    public function jsonSerialize()
    {
        return array(
            'idHistoriqueTable' => $this->idHistoriqueTable,
            'historiqueNumeroTable' => $this->historiqueNumeroTable,
            'historiqueNbPersonne' => $this->historiqueNbPersonne,
        );
    }

}